prov/gni/src/gnix_wait.c

```diff
@@ -277,8 +274,8 @@ int gnix_wait_close(struct fid *wait)
 	}
 
 	if (wait_priv->type == FI_WAIT_FD) {
-		close(wait_priv->fd[WAIT_READ]);
-		close(wait_priv->fd[WAIT_WRITE]);
+		ofi_close_socket(wait_priv->fd[WAIT_READ]);
+		ofi_close_socket(wait_priv->fd[WAIT_WRITE]);
 	}
 
 	_gnix_ref_put(wait_priv->fabric);
```

Comment:
please revert changes to this file.  The gni provider won't support windows anyway.